src/flows/shared/steps/OrganizeCollection/Editor/StagedNftImage.tsx

```diff
@@ -76,63 +79,73 @@ function StagedNftImage({ tokenRef, size, hideLabel, setNodeRef, onLoad, ...prop
 type StagedNftImageImageProps = {
   url: string;
   size: number;
-  tokenName: string | null;
   hideLabel: boolean;
   onLoad: () => void;
-  collectionName: string | null;
+  tokenRef: StagedNftImageImageFragment$key;
   setNodeRef: (node: HTMLElement | null) => void;
 };
 
 function StagedNftImageImage({
   url,
   size,
   onLoad,
+  tokenRef,
   hideLabel,
-  tokenName,
   setNodeRef,
-  collectionName,
   ...props
 }: StagedNftImageImageProps) {
+  const token = useFragment(
+    graphql`
+      fragment StagedNftImageImageFragment on Token {
+        ...NftPreviewLabelFragment
+      }
+    `,
+    tokenRef
+  );
+
   const { handleError } = useThrowOnMediaFailure('StagedNftImageImage');
 
   // We have to use this since we're not using an actual img element
   useImageFailureCheck({ url, onLoad, onError: handleError });
 
   return (
     <StyledGridImage srcUrl={url} ref={setNodeRef} size={size} {...props}>
-      {hideLabel ? null : (
-        <StyledNftPreviewLabel title={tokenName} collectionName={collectionName} />
-      )}
+      {hideLabel ? null : <StyledNftPreviewLabel tokenRef={token} />}
     </StyledGridImage>
   );
 }
 
 type StagedNftImageVideoProps = {
   url: string;
   size: number;
-  tokenName: string | null;
   hideLabel: boolean;
   onLoad: () => void;
-  collectionName: string | null;
+  tokenRef: StagedNftImageVideoFragment$key;
   setNodeRef: (node: HTMLElement | null) => void;
 };
 
 function StagedNftImageVideo({
   url,
   size,
   onLoad,
+  tokenRef,
   hideLabel,
-  tokenName,
   setNodeRef,
-  collectionName,
   ...props
 }: StagedNftImageVideoProps) {
+  const token = useFragment(
+    graphql`
+      fragment StagedNftImageVideoFragment on Token {
+        ...NftPreviewLabelFragment
+      }
+    `,
+    tokenRef
+  );
+
   return (
     <VideoContainer ref={setNodeRef} size={size} {...props}>
       <StyledGridVideo onLoad={onLoad} src={url} />
-      {hideLabel ? null : (
-        <StyledNftPreviewLabel title={tokenName} collectionName={collectionName} />
-      )}
+      {hideLabel ? null : <StyledNftPreviewLabel tokenRef={token} />}
```

Comment:
could we make it so that the collection name doesn't appear clickable in the editor? 

![Screen Shot 2022-09-12 at 17 09 10](https://user-images.githubusercontent.com/80802871/189638511-854f305c-5ad7-492b-8301-175699d7b16e.png)

Reply:
we want the name, not the link (note for terence)